packages/cozy-scanner/src/stylesheet.css

```diff
@@ -0,0 +1,30 @@
+.border-selected {
+  border: 0.063rem solid var(--dodgerBlue);
+  border-radius: 0.25rem;
+}
+.border-not-selected {
+  border: 0.063rem solid var(--silver);
+  border-radius: 0.25rem;
+}
+.grid-item {
+  height: 5.25rem;
+  padding-top: 1rem;
+}
+/*This 19 number is pretty magic. it seems that the qualify icon 
+  is not well centered. So I can't just rem(16) ... */
+.grid-item-title {
+  margin-top: 1.188rem;
+}
+.grid-item__selected__without_label {
+  padding-top: 1rem;
+}
+.grid-item__selected {
+  padding-top: 0.75rem;
+}
+.grid-item-theme {
+  font-size: 0.75rem;
+  padding-top: 0.25rem;
+}
+.grid-item-label {
+  font-size: 0.625rem;
+}
```

Comment:
This does not look specific to the scanner ? Can you add a comment so that we do not forget that this should be in UI ?